t/request-id-err-log.t

```diff
@@ -0,0 +1,192 @@
+use t::APISIX_NGINX 'no_plan';
+
+run_tests;
+
+__DATA__
+
+=== TEST 1: request_id in error log set
```

Comment:
we can use `apisix_request_id` here, we mainly use this name later

Reply:
should the log also have `apisix_request_id` instead of `request_id` appended to the log?
https://github.com/api7/apisix-nginx-module/pull/93/files#diff-8ba7029d7a95ad3485f65df9a8e586b8277556981549bd73b846958090fd088fR20

Reply:
done please review again